javascript/javascript2/week1/week1Js2.js

```diff
@@ -0,0 +1,107 @@
+
+// shortest Word;
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shorterword(danishWords) {
+    let shortestWord;
+    for (let i = 0; i < danishWords.length; i++) {
+        if (danishWords[i].length < danishWords[i++].length) {
+            shortestWord = danishWords[i];
+        }
+        else {
+            shortestWord = danishWords[i++]
+        }
+    }
+    return shortestWord;
+} 
+
+
+console.log(shorterword(danishWords))  // returns 'ø'
+
+///////// Danish Letter Counter
+
+//Find and count the Danish letters
+const danishString = "Jeg har en blå bil";
+
+const danishString2 = "Blå grød med røde bær";
+//danishString2 // returns {total: 4, æ: 1, ø: 2, å: 1}
+
+
+let object = {};
+function DanishLettersCounter(danishString) {
+    let count1 = 0;
+    let count2 = 0;
+    let count3 = 0;
+    let total = 0;
+    for (let i = 0; i < danishString.length; i++) {
+        if (danishString[i] === 'å' || danishString[i] === 'Å') {
+            count1 += 1;
+        }
+        else if (danishString[i] === 'æ' || danishString[i] === 'Æ') {
+            count2 += 1;
+        }
+        else if (danishString[i] === 'ø' || danishString[i] === 'Ø') {
+            count3 += 1;
+        }
+        total = count1 + count2 + count3;
+
+    }
+    return `{total: ${total}, æ:${count2}, ø:${count3}, å:${count1} } `
+
+}
+console.log(DanishLettersCounter(danishString2))
```

Comment:
As the basic requirement , this output need to be printed in HTML using DOM manipulation.